libs/sdk-ui-tests-e2e/cypress/tools/attributeFilterConfig.ts

```diff
@@ -73,4 +80,16 @@ export class AttributeFilterConfiguration {
     getSaveButton() {
         return new Button(".gd-attribute-filter-apply-button__next.s-save");
     }
+
+    cancel() {
+        cy.get(".dropdown-body .cancel-button").click();
+        cy.wait(200);
```

Comment:
why using wait here? if there is any element that can be used to wait to no exist, use it